src/components/bottom-navbar/BottomNavbar.js

```diff
@@ -155,7 +155,8 @@ const BottomNavbar = (props) => {
                                 <span>Logout</span>
                             </>
                         }
-                    </button>
+                    </button> : null}
```

Comment:
This is debatable if you should do this or not(take out the logout button). The logout button clears state on the app, so you can login as someone else. But at the same time, it's arguable how often a user will be switching accounts anyway.

The token expired handler is not graceful anyway, it has a limit of I think 30 minutes and if it's expired while you're doing something you just get an alert saying "token expired please login" or something like that. Then you'd get to a state where token doesn't exist anymore/login button shows up again.